### PR TITLE
fix(StatusChatList): Fixed alignment in left column

### PR DESCRIPTION
**Pull request by alexandraB99**
Needed for https://github.com/status-im/status-desktop/issues/5211

### Checklist

- [x] follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
  - the scope should be the component's name e.g: `feat(StatusListItem): ... `
  - when adding new components, the scope is the module e.g: `feat(StatusQ.Controls): ...`
- [ ] add documentation if necessary (new component, new feature)
- [x] update sandbox app
  - in case of new component, add new component page
  - in case of new features, add variation to existing component page
  - nice to have: add it to the demo application as well
- [x] test changes in both light and dark theme?
- [ ] is this a breaking change?
    - [ ] use the dedicated `BREAKING CHANGE` commit message section
    - [ ] resolve breaking changes in [status-desktop](https://github.com/status-im/status-desktop)
        - [ ] (pre-merge) adapt code to breaking changes
        - [ ] (post-merge) update StatusQ submodule pointer
- [x] test changes in [status-desktop](https://github.com/status-im/status-desktop)

### CODE REVIEW

File: src/StatusQ/Components/StatusChatInfoToolBar.qml [hunk 1]
```diff
@@ -104,5 +102,4 @@ Item {
             }
         }
     }
-
```
Comment (PascalPrecht) [inline src/StatusQ/Components/StatusChatInfoToolBar.qml:107]: Surprised these would affect the alignment of the chatlist!

**Review by PascalPrecht — APPROVED** [review]
Cool!

**Review by noeliaSD — COMMENTED** [review]
I've tested it in `desktop` and still can see the difference between left and right margins. Maybe I'm not testing the right changes here. Are they necessary some other adjustments in the desktop site to close that issue https://github.com/status-im/status-desktop/issues/5211 and are not included in the current PR?

Here a screenshot of my test:
Screenshot 2022-05-18 at 08 20 43

**Review by noeliaSD — APPROVED** [review]

File: src/StatusQ/Components/StatusChatInfoToolBar.qml (hunk 1 above)
Comment (alexandraB99) [reply src/StatusQ/Components/StatusChatInfoToolBar.qml:107]: since ever!

**Pull request merged**